### DISCUSSION TITLE
Linux suddenly not acknowledging all USB drives

### DISCUSSION THREAD

**Discussion by number435398** [Chia-Network/chia-blockchain#8502] [Q&A]
Just in the last few days I reset my Linux miner box and when it rebooted, it no longer likes many of my external USB drives. It claims to view some of them, but when I disconnect some (that it claims it doesn't even see) it still makes the audio noises suggesting its been disconnected, but still won't acknowledge them when I reconnect them. Its as if its suddenly limited in the number of drives it shows. Then when I open the GUI linux miner, it doesn't see the majority of the drives/plots that are acknowledged by Linux. I looked into the log file and it seems to claim issues of "permission denied", but I have changed no permissions.  Anyone else having similar issues?

I'm wondering if this is caused by some sort of update that I allowed it to do that only took effect once I reset the computer. If it is, I would think this may start to affect this community.

**Comment by loppefaaret** [comment]
you need to ensure that the linux OS is mounting the drives correctly, when that is done, make sure that the permissions are correctly set... once that is done, you can check if Chia is loading the plots correctly

**Comment by BrandtH22** [comment]
@number435398 , since the previous message by loppefaaret is marked as the answer it seems this issue has been resolved.
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
